OutSystems/outsystems-datagrid-reactive
# ROU-3482 - [DataGrid] - Client action that returns the average value of select cells

This PR is to add a new API that allows the user to get the average of a cell selection.

### What was done
* Create method that gets the count and value of number, currency and calculated cells selected and calculates the average of it.

### Test Steps
1. Run the API after a selection;
2. Check if the value is the expected;

### Checklist
